tests/modules/goat/taxonsearch/main.nf

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { GOAT_TAXONSEARCH } from '../../../../modules/goat/taxonsearch/main.nf'
+
+//
+// Test with genus name (Canis)
+//
+workflow test_goat_taxonsearch_genus_name {
+
+    input_file = file('NO_FILE')
```

Comment:
So, change this to `[]` instead of `file('NO_FILE')`

Reply:
Done.